src/spikeinterface/sortingcomponents/clustering/tools.py

```diff
@@ -192,3 +192,58 @@ def apply_waveforms_shift(waveforms, peak_shifts, inplace=False):
             aligned_waveforms[mask, -shift:, :] = wfs[:, :-shift, :]
 
     return aligned_waveforms
+
+
+def get_templates_from_clusters(
+    recording,
+    peaks,
+    peak_labels,
+    ms_before,
+    ms_after,
+    svd_model=None,
+    svd_features=None,
+    sparsity_mask=None,
+    **job_kwargs,
+):
+
+    from spikeinterface.core.template import Templates
+    from spikeinterface.core.waveform_tools import estimate_templates
+
+    mask = peak_labels > -1
+    labels = np.unique(peak_labels[mask])
+
+    fs = recording.get_sampling_frequency()
+    nbefore = int(ms_before * fs / 1000.0)
+    nafter = int(ms_after * fs / 1000.0)
+    num_channels = recording.get_num_channels()
+
+    if svd_model is None:
+        templates_array = estimate_templates(
+            recording, peaks, labels, nbefore, nafter, return_scaled=False, job_name=None, **job_kwargs
+        )
+    else:
```

Comment:
I would not this magic if else.

I think I prefer a clear function that do only the second part.
For instance `compute_template_from_svd_and_label()`
Lets do the if else outside this fonction.